### PR TITLE
Shops Controllor Unit Testing

### PR DESCRIPTION
**Pull request by LevinGeorge**
- Have included all the methods within the shop resource.
- Commented out "update method". Was unsure if we were using the factory gem now or later on..
- If we were using the Factory gem.. The code will look sth similar to the pic 
  e

### CODE REVIEW

File: app/models/shop.rb [hunk 1]
```diff
@@ -3,9 +3,10 @@ class Shop < ActiveRecord::Base
   has_many :products, :dependent => :delete_all
   has_many :withdrawals
 
-  validates :name, length: {minimum: 2}, presence: true
-  validates :description, length: {minimum: 10}, presence: true
-  validates :email_id, presence: true
+
+  validates :name, length: {minimum: 2}, presence:true
+  validates :description, length: {minimum: 75}, presence:true
```
Comment (danielpaul) [inline app/models/shop.rb]: Keep my changes on this.

File: spec/controllers/shops_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        user = create(:user)
+        sign_in user
+        shop_attributes = attributes_for(:shop)
+        expect{
+          post :create, {shop: shop_attributes}
+        }.to change(Shop,:count).by(1)
+      end
+    end
+
+    context "unlogged user" do
```
Comment (danielpaul) [inline spec/controllers/shops_controller_spec.rb]: Call it "logged out user".

File: spec/controllers/shops_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+RSpec.describe ShopsController, type: :controller do
+
+  # describe "GET #index" do
```
Comment (danielpaul) [inline spec/controllers/shops_controller_spec.rb]: A lot of these tests are commented out?

File: app/models/shop.rb (hunk 1 above)
Comment (starkshaw) [inline app/models/shop.rb]: You didn't inform us. We all keep it 75.

Comment (LevinGeorge) [inline app/models/shop.rb]: :+1:

**Pull request merged**